### DISCUSSION TITLE
Chrome shows different result for modal custom event

### DISCUSSION THREAD

**Discussion by munaja** [twbs/bootstrap#36181] [Q&A]
Before create an issue, I think it's better to discuss here, in case i am the one missing something.

I tried the `show.bs.modal` event provided bootstrap modal. The following code is basically from the documentation page 
```
var myModalEl = document.getElementById('exampleModal');
myModalEl.addEventListener('show.bs.modal', function (event) {
	console.log(event); // showing what is in the event
})
```

Firefox shows expected result where I can get information about the source of the element I click (the trigger), it is in the `event.explicitOriginalTarget`. But Chrome missing attribute `explicitOriginalTarget` from the `event`.  Actually Chrome missing some other attribute like : `originalTarget` and `relatedTarget`.

Is it a bug or am I missing something?

Additional information, here is the button that toggles the modal. I put it inside a dropdown item.
```
<button class="dropdown-item" data-bs-toggle="modal" data-bs-target="#exampleModal" data-id="1" data-action="approve">
  <i class="bi bi-pencil me-2"></i>Accept
</button>
```

EDIT :
I still can get the information about clicked element `event.relatedTarget`

**Comment by GeoSot** [comment]
The `explicitOriginalTarget ` is not a standard property, according to   https://developer.mozilla.org/en-US/docs/Web/API/Event/explicitOriginalTarget

However, is my intention to add some custom information on BS events, but this will happen at least on 5.3

**Reply by munaja** [reply]
And regarding `event.relatedTarget` I mentioned before, is it also not standard?

I forgot to mention that from `console.log(event)`, Firefox shows `relatedTarget` as one of the attribute, but in chrome it doesn't show up, but I still can access it. Currently it is the only thing that helps me to know which element triggers the modal to pop up, thanks to it.
